ui/src/App.tsx

```diff
@@ -27,20 +27,11 @@ function App(): JSX.Element {
               <ShoeMap />
             </Route>
             <Route exact path="/login">
-              <Login login />
+              <Login />
```

Comment:
Is there a variable I'm missing? Getting this in my make error:
![image](https://user-images.githubusercontent.com/19617248/114284287-cd5b3300-9a1c-11eb-806c-316c07758ab4.png)

Also this is expected behaviour for `/login` and `admin` - my blueprint email should be an invalid sign in?
![bcd47521-aea2-41de-a27b-34cbc6e1bdfe](https://user-images.githubusercontent.com/19617248/114284411-abae7b80-9a1d-11eb-8d32-287e1436e4b6.gif)

Reply:
Actually wait that shouldn't be expected behaviour because it looks like `auth.go` should validate uwblueprint.org emails.

Reply:
Huh weird @nimwijetunga might have more context here.

Reply:
@hanlinc27 copy the newest env file from pinned to the shoe project channel. The error your getting is because you have a wrong client id

Reply:
ooh yea you're right, it works now we good